resist flex shrinkage on the circle control

dschuyler opened the pull request:
Fixes #114 
@keanulee

dschuyler committed:
resist flex shrinkage on the circle control

Fixes #114

keanulee commented on paper-radio-button.html:
```diff
@@ -74,6 +74,7 @@
       #radioContainer {
         @apply(--layout-inline);
         @apply(--layout-center-center);
+        flex-shrink: 0;
```
By default, `#radioContainer`'s parent (`:host`) is not a flex container, so unless you're making it one (e.g. `paper-radio-button { display: flex }`), `#radioContainer` should not shrink.

dschuyler committed:
Changed flex-shrink to apply mixin

While there are no known issues with using the flex-shrink here, it's more general to use a mixin (and the small chance of an issue coming up becomes far smaller).
